mantidproject/mantid
# LoadPSIMuonBin Algorithm Improvements

**Description of work.**
PSI temperature files can now be read-in alongside the workspace loadable files to and added to the sample logs of the created workspace. Alongside these changes, several sample logs are now numeric instead of strings.

**Report to:** Francis Pratt (ISIS)

**To test:**
- Load `deltat_tdc_dolly_1529.bin` from the Unit Test data
- Open the sample logs
- Check the sample logs with the names that start with `Temp_` and make sure that the temperature values against time make a plot, (If none of the number series appear, build the test data).
- Do some testing around with the algorithm looking for files manually and searching for them etc.



Fixes #23070, do not close the issue.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
